README.md

```diff
@@ -13,13 +13,35 @@ To build and install the latest version in a running AEM instance, simply do
 ```
 mvn clean install sling:install
 ```
+
 This installs everything by default to `localhost:4502` without any context path. You can also configure the install location with the following maven properties:
-* `aem.host`: the name of the AEM instance
-* `aem.port`: the port number of the AEM instance
-* `aem.contextPath`: the context path (if any) of your AEM instance, starting with `/`
+
+-   `aem.host`: the name of the AEM instance
+-   `aem.port`: the port number of the AEM instance
+-   `aem.contextPath`: the context path (if any) of your AEM instance, starting with `/`
 
 ## Using the GraphQL client
 
+### Prerequisites
+
+The GraphQL client needs a service user to read the configurations, so consumers must provide one or more user mappings. The subservice name is `graphql-client-configuration` and the minimum privilege matrix for the service user(s) is the following:
+
+| **Path**       | **Privilege** |
+| -------------- | ------------- |
+| /conf          | jcr:read      |
+| /content       | jcr:read      |
+| /libs/settings | jcr:read      |
+
+You can either use one service user or an aggregate of several users, each with its own privilege set. The following mapping would work in a standard AEM installation:
+
+```
+[com.adobe.commerce.cif.connector-graphql:omnisearch-service=omnisearch-service]
```

Comment:
I think you should be more precise here. I personally didn't understand if this is just a simple example, or if we really recommend customers to use the `omnisearch-service` user because it has the right permissions.